tools/dev/msftidy.rb

```diff
@@ -196,6 +196,8 @@ def check_ref_identifiers
             warn("Please use 'US-CERT-VU' for '#{value}'")
           elsif value =~ /^https?:\/\/wpvulndb\.com\/vulnerabilities\//
             warn("Please use 'WPVDB' for '#{value}'")
```

Comment:
So looks like this is only used in https://github.com/rapid7/metasploit-framework/blob/c0b42ff7a2a1b677aaba6acd7ebeefddcffdf38e/documentation/modules/auxiliary/admin/http/wp_gdpr_compliance_privesc.md which is a documentation file and not a module. It may be an idea to remove this.

I'd also be curious if other files need updating according to https://github.com/rapid7/metasploit-framework/search?q=wpvulndb.com&type=code. Seems there is also https://github.com/rapid7/metasploit-framework/blob/85a9accbeec063f52406621f124a7f82ad63dcba/lib/msf/core/module/reference.rb#L114 which relies on the old URL.